tests/src/common/WskCli.java

```diff
@@ -637,7 +637,7 @@ public RunResult invoke(int expectedCode, String name, Map<String, String> param
      *
      * @param name
      *            the name of the action.
-     * @param expecte
+     * @param expected
      *            dthe expected activation id
```

Comment:
Yes. It was in the list of words for me to look at.

I was considering 2324 words and apparently didn't check that one.

Basically, my tool calculates things that are tokens and excludes things it knows are words, and then leaves me to decide which are probably typos vs. which are probably intentional identifiers (ansible)/actual linenoise (AQAABAAAAABQ).